### PR TITLE
Added more information about how get_node_light works.

### PR DESCRIPTION
**Pull request by RobertZenz**

### CODE REVIEW

**Comment by RobertZenz** [comment]
Originated in #3149.

File: doc/lua_api.txt [hunk 1]
```diff
@@ -1914,9 +1914,14 @@ and `minetest.auth_reload` call the authetification handler.
     * Returns `{name="ignore", ...}` for unloaded area
 * `minetest.get_node_or_nil(pos)`
     * Returns `nil` for unloaded area
-* `minetest.get_node_light(pos, timeofday)` returns a number between `0` and `15` or `nil`
+* `minetest.get_node_light(pos, timeofday)`
+    * Gets the light value at the given position. Note that if the given
+      position is a node, the light value "inside" the node will be returned.
+      So if you want to know the light level of a certain node, you need to
```
Comment (rubenwardy) [inline doc/lua_api.txt]: `certain non-air node`?

**Comment by PilzAdam** [comment]
> Note that if the given position is a node

The position is always a node. Also the other part is misleading, since the top position is not always the correct light value (e.g. at walls or ceilings).
Suggestion:
```
Note that the light value "inside" the node at pos is returned, so you usually want the light value of a neighbor.
```

**Comment by est31** [comment]
:+1: to @PilzAdam 's suggestion.

File: doc/lua_api.txt (hunk 1 above)
Comment (est31) [inline doc/lua_api.txt]: There are non-air nodes that are transparent as well, like grass or glass. I think the criterion is if you set the drawtype to light.

**Comment by RobertZenz** [comment]
Changed.

**Comment by est31** [comment]
Merged fe6575b8d38b29026adb9bf3d6226b5b3ceb4cab

**Pull request closed without merging**